agent/command/git_test.go

```diff
@@ -461,22 +461,25 @@ func (s *GitGetProjectSuite) TestBuildHTTPCloneCommand() {
 	cmds, err := opts.buildHTTPCloneCommand(false)
 	s.NoError(err)
 	s.Require().Len(cmds, 5)
-	s.Equal("set +o xtrace", cmds[0])
-	s.Equal("echo \"git clone https://[redacted oauth token]:x-oauth-basic@github.com/evergreen-ci/sample.git 'dir' --branch 'main' --filter=tree:0 --single-branch\"", cmds[1])
-	s.Equal("git clone https://PROJECTTOKEN:x-oauth-basic@github.com/evergreen-ci/sample.git 'dir' --branch 'main' --filter=tree:0 --single-branch", cmds[2])
-	s.Equal("set -o xtrace", cmds[3])
-	s.Equal("cd dir", cmds[4])
-
+	s.True(utility.ContainsOrderedSubset([]string{
```

Comment:
nit: this test seems like it pretty much boils down to checking for string slice equality. Is this test interested in exact equality of the output, or just subset ordering? If it's the latter, then this is fine as-is, but if it's the former, it should just do something like:
```go
s.Equal([]string{
    "set +o trace",
    "echo \"git clone https://[redacted oauth token]:x-oauth-basic@github.com/evergreen-ci/sample.git 'dir' --branch 'main' --filter=tree:0 --single-branch\""
    /* + the others... */
    }, cmds)
```

Reply:
It's a good question. My thinking was the tests are just testing if it compiles a command list that does contain that command - disregarding the other commands (as they are tested in some of the other functions). That the intention of this test was to make sure it correctly compiles this part

Reply:
Sounds good. Thanks!